src/main/java/com/demo/model/Driver.java

```diff
@@ -0,0 +1,38 @@
+package com.demo.model;
+
+import lombok.Data;
+import org.springframework.stereotype.Component;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+
+@Component
```

Comment:
potrzebne?

Reply:
Nie, nie jest to potrzebne. Wystarczy adnotacja @Entity